ludwig/utils/schema_utils.py

```diff
@@ -159,6 +223,74 @@ def _jsonschema_type_mapping(self):
         }
 
 
+def IntegerOrStringOptionsField(
+    options: List[str],
+    nullable: bool,
+    default: Union[None, int],
+    isIntegeric: bool = True,
```

Comment:
nit: I'm not sure Integeric is a word. Maybe "is_like_integer", "is_integer_like", "is_integer"?

Reply:
Haha I was thinking of [this](https://www.npmjs.com/package/is-integeric), but sure - fixed